_includes/lesson_footer.html

```diff
@@ -5,22 +5,22 @@
   <div class="row">
     <div class="col-md-6 copyright" align="left">
 	{% if site.carpentry == "swc" %}
-	Copyright &copy; 2018–{{ 'now' | date: "%Y" }}
+	<a href="{{ site.cc_by }}">CC-BY</a>  &copy; 2018–{{ 'now' | date: "%Y" }}
```

Comment:
I think the use of `&copy'` here is inappropriate, see https://creativecommons.org/choose/.

Reply:
I think Raniere is right: it should be something like
```
The Carpentries &copy; 20XX - 20XX
CC-BY...
```